wordpress-mobile/gutenberg-mobile
# Fix crash once adding a Group block

Fixes crash once adding `Group` block

Ref to gutenberg PR: https://github.com/WordPress/gutenberg/pull/19457

To test:

1. Add a Group block
2. Check if block is added correctly

PR submission checklist:
